src/travel-places/transport-air/1F680.svg

```diff
@@ -1,30 +1,30 @@
-<svg id="emoji" x="0" y="0" viewBox="0 0 72 72" version="1.1" xmlns="http://www.w3.org/2000/svg">
+<svg id="emoji" xmlns="http://www.w3.org/2000/svg" enable-background="new 0 0 72 72" xml:space="preserve" viewBox="0 0 72 72" height="72" width="72" version="1.1" y="0" x="0">
```

Comment:
can you do this? 

`node_modules/.bin/svgo src/activities/arts-crafts/1F3A8.svg --config helpers/beautify-svg.yml` 

it will remove `xml:space="preserve"` and other things that are unnecessary